nixpkgs_review/cli/__init__.py

```diff
@@ -194,6 +194,7 @@ def parse_args(command: str, args: List[str]) -> argparse.Namespace:
     comments_parser.set_defaults(func=show_comments)
 
     merge_parser = subparsers.add_parser("merge", help="merge PR")
+    merge_parser.add_argument("-m", "--method", default="merge", help="Merge method")
```

Comment:
Did you forget to commit a file? `merge_parser.method` is not used anywhere.

Reply:
Fixing it soon. Setting WIP for now!

Reply:
Ok sorry, for the long wait. Should be fixed now.